docs/content/en/locale-fallback.md

```diff
@@ -1,13 +1,15 @@
 ---
-title: Locale fallback description: "How a fallback gets selected when a translation is missing"
-position: 11 category: Guide
+title: Locale fallback
+description: "How a fallback gets selected when a translation is missing"
+position: 11
+category: Guide
 ---
 
 
-nuxt-i18n takes advantage of vue-i18n ability to handle localization fallback. It is possible to define a single fallback locale, an array of locales,
+**nuxt-i18n** takes advantage of **vue-i18n** ability to handle localization fallback. It is possible to define a single fallback locale, an array of locales,
 or a decision map for more specific needs.
 
-```js {}[nuxt.config.js]
+```js{}[nuxt.config.js]
```

Comment:
Please add back a space.

Reply:
Are you sure? I've seen no reference with a space. All these occurrences have no space.

Reply:
There should be a space after `js` as otherwise, some editors won't recognize the block as using JS syntax. It will work either way for docs.

Can also skip `{}`.

Reply:
Do you want me to replace them all? It could be the occasion.

Reply:
Feel free if you don't mind. :)

Reply:
No problem :)

Reply:
Oh crap. I forgot that I've done that yesterday myself. I have too many repos to keep track of. ;)

Maybe just revert your commit then and just apply in the ones you've added.

Reply:
Job's done :)